packages/core/content-manager/server/src/services/entity-manager.ts

```diff
@@ -364,13 +367,18 @@ export default ({ strapi }: { strapi: Strapi }) => ({
     return mappedEntity;
   },
 
-  async countDraftRelations(id: string, uid: Common.UID.ContentType) {
+  async countDraftRelations(id: string, uid: Common.UID.ContentType, locale: string = 'en') {
```

Comment:
the default locale could be left as empty, as the document service will be the one filling it with the default language of the app (which might not be english)